### PR TITLE
Move yum.cache to disk after partitioning (#1287438)

### PR DESCRIPTION
**Pull request by bcl**
In some installations the size of yum.cache can exceed that of the
tmpfs used for /tmp/yum.cache or even that of the available RAM so move
it to the disk after partitioning has finished.

Resolves: rhbz#1287438

### CODE REVIEW

**Comment by jkonecny12** [comment]
Looks good to me.

File: pyanaconda/packaging/yumpayload.py [hunk 1]
```diff
@@ -400,12 +400,17 @@ def _writeInstallConfig(self):
             except PayloadSetupError as e:
                 log.error(e)
 
-        releasever = self._yum.conf.yumvar['releasever']
-        self._writeYumConfig()
+        # Move the yum.cache to the new disk space to save memory
+        new_cache = iutil.getSysroot()+"/var/tmp/yum.cache"
+        os.makedirs(iutil.getSysroot()+"/var/tmp")
+        if execWithRedirect("mv", [_yum_cache_dir, new_cache]) > 0:
```
Comment (vpodzime) [inline pyanaconda/packaging/yumpayload.py]: Any reason why not to use ``shutil.move()`` here?

**Comment by bcl** [comment]
> Any reason why not to use shutil.move() here?

Good idea.

**Comment by bcl** [comment]
Pushed.

**Pull request closed without merging**